website/docs/docs/cloud/secure/snowflake-privatelink.md

```diff
@@ -19,26 +19,37 @@ Users connecting to Snowflake using SSO over a PrivateLink connection from dbt C
 - [Snowflake SSO with Private Connectivity](https://docs.snowflake.com/en/user-guide/admin-security-fed-auth-overview#label-sso-private-connectivity)
 :::
 
-## Configure PrivateLink
+## About private connectivity for Snowflake
 
-1. Open a Support case with Snowflake to allow access from the dbt Cloud AWS account
-- Snowflake prefers that the account owner opens the Support case directly, rather than dbt Labs acting on their behalf. For more information, refer to [Snowflake's knowledge base article](https://community.snowflake.com/s/article/HowtosetupPrivatelinktoSnowflakefromCloudServiceVendors)
+dbt Cloud supports private connectivity for Snowflake using one of the following services:
+
+- AWS [PrivateLink](#configure-aws-privatelink)
+- Azure [Private Link](#configure-azure-private-link)
+
+## Configure AWS PrivateLink
+
+To configure Snowflake instances hosted on AWS for [PrivateLink](https://aws.amazon.com/privatelink):
+
+1. Open a Support case with Snowflake to allow access from the dbt Cloud AWS or Entra ID account.
```

Comment:
sugg for consistency on page: this instance is not referring to the "Support team", just a support ticket

```suggestion
1. Open a support case with Snowflake to allow access from the dbt Cloud AWS or Entra ID account.
```